cmake/version.properties.in

```diff
@@ -0,0 +1,2 @@
+VERSION_CODE=@APK_VERSION_CODE@
```

Comment:
we had a APP_VERSION_CODE variable (in docker-build), can we uniform the names (APK vs APP)

Reply:
and it seems never defined in docker-build